lib/workers/pr/changelog.js

```diff
@@ -14,6 +14,9 @@ async function getChangeLogJSON(depName, fromVersion, newVersion) {
   if (!fromVersion || fromVersion === newVersion) {
     return null;
   }
+  if (depName === 'node') {
```

Comment:
Could this be an issue if someone is trying to upgrade their [`node`](https://www.npmjs.com/package/node) dependency?

Reply:
Not currently because that package does not publish any release notes. This short-circuit saves us from unnecessarily querying that package's GitHub releases until later when proper release notes are added for this, from the original node repo.

Reply:
Someone could still benefit from a PR containing the list of commits made to the `node` package's [repository](https://github.com/aredridel/node-bin-gen/commits/master).